timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -455,9 +455,7 @@ public void stressTestForPaxosEndpoints() {
     }
 
     private static void assertNumberOfThreadsReasonable(int startingThreads, int threadCount, boolean nonLeaderDown) {
-        // TODO (jkong): Lower the amount over the threshold. This needs to be slightly higher for now because of the
-        // current threading model in batch mode, where separate threads may be spun up on the autobatcher.
-        int threadLimit = startingThreads + 800;
+        int threadLimit = startingThreads + 1000;
         if (nonLeaderDown) {
             if (threadCount > threadLimit) {
```

Comment:
Drive by comment here, but do we really need this if statement?

Reply:
We do not. Look at line 463 😅  This must have been an artifact of when I did some previous testing

Reply:
yep, that's what I was looking at ^^. I'm guessing this was for a breakpoint.